Antidote/PauseCallTableViewCell.m

```diff
@@ -0,0 +1,123 @@
+//
+//  PauseCallCellTableViewCell.m
+//  Antidote
+//
+//  Created by Chuong Vu on 7/16/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+#import "PauseCallTableViewCell.h"
+#import "AppearanceManager.h"
+#import "NSString+Utilities.h"
+#import "Masonry.h"
+
+static const CGFloat kEndCallButtonWidth = 40.0;
+static const CGFloat kNickNameFontSize = 16.0;
+static const CGFloat kCallDurationFontSize = 11.0;
+
+@interface PauseCallTableViewCell ()
+
+@property (strong, nonatomic) UIButton *endCallButton;
+@end
+
+@implementation PauseCallTableViewCell
+
+#pragma mark - LifeCycle
+
+- (id)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSString *)reuseIdentifier
+{
+    self = [super initWithStyle:UITableViewCellStyleSubtitle reuseIdentifier:reuseIdentifier];
+
+    if (! self) {
+        return nil;
+    }
+
+    self.backgroundColor = [UIColor blackColor];
+
+    [self createResumeCallButton];
+    [self createEndCallButton];
+    [self setupFont];
+    [self installConstraints];
+
+    return self;
+}
+
+#pragma mark - View setup
+
+- (void)createEndCallButton
+{
+    UIImage *image = [UIImage imageNamed:@"call-accept"];
+
+    self.endCallButton = [UIButton buttonWithType:UIButtonTypeCustom];
+    [self.endCallButton setBackgroundImage:image forState:UIControlStateNormal];
+    self.endCallButton.tintColor = [UIColor whiteColor];
+    self.endCallButton.backgroundColor = [UIColor redColor];
+    self.endCallButton.layer.cornerRadius = kEndCallButtonWidth / 2.0;
+
+    [self.endCallButton addTarget:self action:@selector(endCallButtonTapped) forControlEvents:UIControlEventTouchUpInside];
+
+    [self.contentView addSubview:self.endCallButton];
+}
+
+- (void)createResumeCallButton
+{
+    UIImage *image = [UIImage imageNamed:@"call-pause"];
+
+    [self.imageView setImage:image];
+    self.imageView.tintColor = [UIColor whiteColor];
```

Comment:
Yea haha. Realized halfway through that I didn't need a button when I could just use the imageView. Will change it.